cmake/Findnl-3.cmake

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+find_library(NL3_LIBRARY "libnl-3.so")
+find_path(NL3_INCLUDE_DIRS libnl3/netlink/netlink.h)
```

Comment:
I think its better to use PATH_SUFFIXES and NAMS instead of the entire path 

find_path(NL3_INCLUDE_DIRS 
    NAMES netlink.h 
    PATH_SUFFIXES libnl3
)

Reply:
Exactly. PATH_SUFFIXES makes sure that it is set to /usr/include/libnl3, like it should be instead of /usr/include.

NAMES is not needed since we look for only one. And the file name should include the netlink part, because we do `#include <netlink/netlink.h>`. So:

```
find_path(NL3_INCLUDE_DIRS netlink/netlink.h PATH_SUFFIXES libnl3)
```